README.md

```diff
@@ -140,6 +141,37 @@ A detection mode is available in order to avoid interruptions of a running clust
 
 To activate the detection mode, set the `detection_mode` flag to `true` in `helm/values.yaml`.
 
+### Alerting
+
+You can receive notification on the admission decisions made by Connaisseur on basically every REST endpoint that accepts JSON payload. For Slack, Opsgenie and Keybase we have preconfigured payloads that are ready to use, but you can also use the existing payload templates as an example how to create your own custom one. It is also possible to configure multiple different interfaces you want to receive alerts at for each event category (admit/deny) in the `helm/values.yaml`.
+
+If you for example would like to receive notifications in Keybase whenever Connaisseur admits a request to your cluster, your alerting configuration would look similar to to following snippet:
+
+
+```
+alerting:
+  admit_request:
+    templates:
+      - template: keybase
+        receiver_url: https://bots.keybase.io/webhookbot/IFP--tpV2wBxEP3ArYx4gVS_B-0
```

Comment:
This looks like a secret that is here accidentally. I know it's randomly generated, but it may confuse people and security tooling. I would suggest to prob a placeholder instead, such as `webhookbot/Your-Hook-Token`. Also applies to other places.

Reply:
Changed it :)